elyra/cli/pipeline_app.py

```diff
@@ -300,5 +301,66 @@ def run(pipeline_path):
     print_banner("Elyra Pipeline Local Run Complete")
 
 
+@click.command()
+@click.option('--json',
+              'json_option',
+              is_flag=True,
+              required=False,
+              help='Returns pipeline file in a machine-readable JSON format')
+@click.argument('pipeline_path')
+def describe(json_option, pipeline_path):
+    """
+    Display pipeline description in a human readable format
+    """
+
+    _validate_pipeline_file_extension(pipeline_path)
+
+    pipeline_definition = \
+        _preprocess_pipeline(pipeline_path, runtime='local', runtime_config='local')
+
+    indent_length = 4
+    for current_pipeline in pipeline_definition["pipelines"]:
+        pipeline_keys = ["name", "description", "type", "version", "nodes", "dependencies"]
+
+        list_keys = {"dependencies"}
+
+        describe_dict = OrderedDict()
+
+        properties = current_pipeline["app_data"]["properties"]
+
+        pipeline_data = current_pipeline["app_data"]
+
+        # If the name is actually "None", it will seem as if there is no name
+        # The same can be said for all fields
+        describe_dict["name"] = str(properties.get("name", "None"))
+
+        describe_dict["description"] = str(properties.get("description", "None"))
+
+        describe_dict["type"] = str(properties.get("runtime", "None"))
+
+        describe_dict["version"] = str(pipeline_data.get("version", "None"))
+
+        describe_dict["nodes"] = str(len(current_pipeline.get("nodes", [])))
+
+        for node in current_pipeline["nodes"]:
+            if "dependencies" in node["app_data"]["component_parameters"]:
+                for dependency in node["app_data"]["component_parameters"].get("dependencies", []):
+                    if describe_dict.get("dependencies") is None:
+                        describe_dict["dependencies"] = []
+                    describe_dict["dependencies"].append(f"{dependency}")
+        if not json_option:
+            for key in pipeline_keys:
+                if key in list_keys:
+                    click.echo(f"{key.title()}")
```

Comment:
I think these should have a trailing `:` as well.
```suggestion
                    click.echo(f"{key.title()}:")
```